docker-hub/roles-and-permissions.md

```diff
@@ -0,0 +1,76 @@
+---
+description: Use roles in your organization to control who has access to content, registry, and organization management permissions.
+keywords: members, teams, organization, company, roles, access
+title: Roles and permissions
+---
+
+Organization and company owners can assign roles to individuals giving them different permissions in the organization. This section is for owners who want to learn about the defined roles and their permission scopes.
+
+## Roles
+
+When you invite users to your organization, you assign a role. A role is a collection of permissions. Roles define access to perform actions like creating repositories, pulling images, creating teams, and configuring organization settings.
+
+The following roles are available to assign:
+
+- **Member** - Non-administrative role. Members can view other members that are in the same organization.
+- **Editor** - Partial administrative access to the organization. Editors can create, edit, and delete repositories. They can also edit an existing team's access permissions.
+- **Organization owner** - Full organization administrative access. Organization owners can manage organization repositories, teams, members, settings, and billing.
+- **Company owner** - In addition to the permissions of an organization owner, company owners can configure settings for their associated organizations.
+
+Owners can manage roles for members of an organization on [Docker Hub](/docker-hub/members/#update-a-member-role), as well members of an [organization](/admin/organization/members/#update-a-member-role) or a [company](/admin/company/users/#update-a-member-role) on [Docker Admin](/admin/).
+
+## Permissions
+
+The following sections describe the permissions for each role.
+
+### Content and registry permissions
+
+The following outlines content and registry permissions for member, editor, and organization owner roles. Company owners have the same access as organization owners for all associated organizations. See [Company overview](/admin/company/).
+
+| Permission | Member | Editor | Organization owner |
+|:----------------------- |:------ |:-------|:------------------ |
+| Explore images and extensions | ✅ | ✅ | ✅ |
+| Star, favorite, vote, and comment on content | ✅ | ✅ | ✅ |
+| Pull images | ✅ | ✅ | ✅ | ✅ |
+| Create and publish an extension | ✅ | ✅ | ✅ | ✅ |
+| Become a Verified, Official, or Open Source publisher | ❌ | ❌ | ✅ |
+| Observe content engagement as a publisher | ❌ | ❌ | ✅ |
+| Create public and private repositories | ❌ | ✅ | ✅ |
+| Manage, including delete, a repository | ❌ | ✅ | ✅ |
```

Comment:
Suggest removing this line: Members can do this -> it's a team permission

Reply:
I reworded this line slightly based on some discussion with Accounts team, since this is a main feature of the editor role. @siarheikarneichyk what do you think about removing this row and the following ones in the next comments? Or does it just need to be clearer that these are default permissions before a member is added to a team?

Reply:
I'm unsure about `Become a Verified, Official, or Open Source publisher`and `Observe content engagement as a publisher` because I barely remember how it works. The registry team should know better. The most important to keep `Create public and private repositories`. Editors can `Manage, including delete, a repository` but members that are not part of the team with admin permissions can't so I would keep it.

Reply:
Members that are granted team permissions to admin a repo can definitely "manage" a repository, depending on what you consider "managing" -> they can configure builds, webhooks, delete tags, etc. They might even be able to delete - so we at least should remove `manage`

Reply:
Would `Edit and delete repositories` be better? That is how this role is described in the UI when owners are viewing the roles to assign. I can also add to the paragraph below this table: `When you add members to a team, you can manage their repository permissions.` This is followed by a link to the team permissions docs. Would that make it more clear that members can do these things once added to a team?

Also @jcarter3 could you weigh in on whether the publisher permissions are accurate? Thank you!

Reply:
[Sign-off in Slack](https://docker.slack.com/archives/C05CJ2284UR/p1690389111047939?thread_ts=1689889125.642549&cid=C05CJ2284UR) so marking this resolved, am also happy to update the table in a fast-follow PR after today's release if needed.

Reply:
I think we should make clear that these permissions and roles apply to the whole organization and all the repositories in the namespace of the organization. Also indicate that there are other permissions at the repository level that are granted through teams, and those permissions remain independent but work together with roles in order to determine the correct level of access for a user.

Maybe also provide some examples of different scenarios, like:

> A user X that has member role in the org O but is also member of a team T that has admin permissions over a repository R, the result in this case is that the user X will have admin privileges over that specific repository R but not over any other repo in the org O

Or:

> A user X that has editor role in the org O, the result will be that user X will have admin privileges over all repositories in the namespace of the org O, even if X is not a member of any team in the org"

Reply:
I can add to the intro before the first table about permissions applying to the entire org and all repos in the namespace of the org. There's a section that links to setting up repo-specific permissions. I like the idea of examples but I think they might need a visual like a diagram to be clear. I'll make a ticket for that as a follow-up item. Thanks @ivan-californias